src/detext/train/train.py

```diff
@@ -298,6 +303,16 @@ def model_fn(features, labels, mode, params):
 
     group_size_field = features['group_size'] if mode != tf.estimator.ModeKeys.PREDICT else None
 
+    # For multitask training
+    task_id_field = features.get('task_id', None)
+
+    # Override the default weight for multitask from input params task_ids and task_weights
+    if params.task_ids is not None:
+        weight = tf.zeros(shape=tf.shape(weight), dtype="float32")
+        for task_id, task_weight in params.task_ids.items():
```

Comment:
Before reading this line, I don't know the item of params.task_ids is task_weight. Can you have a better name for the variable?

Reply:
Thanks for the suggestion. I'll update with a better name and avoid overriding the input param `task_ids`.

Reply:
Added a new param `task_id_weight_map` that zips task_id and weight into a map.